DQM/L1TMonitorClient/python/L1TEMTFMonitorClient_cff.py

```diff
@@ -0,0 +1,34 @@
+import FWCore.ParameterSet.Config as cms
+
+# L1 Trigger client DQM sequence
+#
+# used by DQM GUI: DQM/Integration/python/test/l1t_dqm_sourceclient-*_cfg.py
+#
+# standard RawToDigi sequence must be run before the L1T module, labels 
+# from the standard sequence are used as default for the L1 DQM modules
+# any configuration change in the unpacking must be done in l1t_dqm_sourceclient-*_cfg.py
+#
+# see CVS for previous authors
+#
+# V.M. Ghete 2011-05-26 revised version of L1 Trigger client DQM
+#                       
+
+# DQM quality tests 
+from DQM.L1TMonitorClient.L1TStage2EMTFQualityTests_cfi import *
+
+# L1 event info DQM client 
+from DQM.L1TMonitorClient.L1TEMTFEventInfoClient_cfi import *
```

Comment:
This could be loaded in DQM/L1TMonitorClient/python/L1TStage2MonitorClient_cff.py (after line 20 for example) and l1tEMTFEventInfoClient could be put directly in the l1TStage2Clients sequence there.